eth/api_backend.go

```diff
@@ -276,7 +276,15 @@ func (b *EthAPIBackend) Downloader() *downloader.Downloader {
 }
 
 func (b *EthAPIBackend) SuggestPrice(ctx context.Context) (*big.Int, error) {
-	return b.gpo.SuggestPrice(ctx)
+	return b.gpo.SuggestPrice(ctx, false)
+}
+
+func (b *EthAPIBackend) SuggestTip(ctx context.Context) (*big.Int, error) {
+	return b.gpo.SuggestPrice(ctx, true)
+}
+
+func (b *EthAPIBackend) SuggestFeeCap(ctx context.Context) (*big.Int, error) {
+	return new(big.Int).Mul(b.CurrentBlock().BaseFee(), common.Big2), nil
```

Comment:
If the network is congested, then at the worst case, the transaction can't be included after 6 blocks.

```python
>>> 1.125 ** 6
2.0272865295410156
```

I'm not sure what's the best factor here. Perhaps we can use the `gpo` for some congestion information. For example if all the previous blocks are fulled with transactions, then this number can be a bit larger.

But for the first version, this hard coded number LGTM. Just throw out the thoughts here.

Reply:
It seems like this is a good direction and something that @MicahZoltu brought up here: https://github.com/ethereum/eth1.0-specs/pull/201#issuecomment-851113884

Reply:
Recommend reading the linked discussion for full context, but my preference is that Ethereum clients don't suggest a fee cap to users because users will use it out of habit instead of changing their behavior to what is optimal for them and the network, which is to set the fee cap to their willingness to pay.